src/seedu/addressbook/commands/Command.java

```diff
@@ -70,4 +70,14 @@ public int getTargetIndex() {
     public void setTargetIndex(int targetIndex) {
         this.targetIndex = targetIndex;
     }
+
+    /**
+     * Checks if command mutates data in the address book.
```

Comment:
Good job remembering to add a header comment to this method. 👍
It can be shortened as follows without losing any info.

/**
  * Returns true if the command potentially mutates data
  */